pkg/services/sqlstore/user.go

```diff
@@ -818,6 +822,97 @@ func UserDeletions() []string {
 	return deletes
 }
 
+func UserUpdates() []string {
+	deletes := []string{
+		"UPDATE star set email = ?, user_id = ?, login = ? WHERE user_id = ?",
+		"UPDATE team_member set user_id = ? WHERE user_id = ?",
+		"DELETE FROM team_member WHERE user_id = ?",
+		"UPDATE FROM " + dialect.Quote("user") + " WHERE id = ?",
+		"UPDATE FROM org_user WHERE user_id = ?",
+		"DELETE FROM dashboard_acl WHERE user_id = ?",
+		"DELETE FROM preferences WHERE user_id = ?",
+		"DELETE FROM team_member WHERE user_id = ?",
+		"DELETE FROM user_auth WHERE user_id = ?",
+		"DELETE FROM user_auth_token WHERE user_id = ?",
+		"DELETE FROM quota WHERE user_id = ?",
+	}
+	return deletes
+}
+
+func (ss *SQLStore) updateUserIds(intoUser user.User, fromUser user.User, sess *DBSession) error {
+	// TODO:
+	/*
+		tables that have user_id references
+
+			tbl_name |
+			--- |
+			temp_user |
+			star |
+			org_user |
+			dashboard_snapshot |
+			quota |
+			preferences |
+			annotation |
+			team_member |
+			dashboard_acl |
+			user_auth |
+			user_auth_token |
+			user_role |
+			query_history_star |
+
+			tables take using this query
+			```sql
+			select tbl_name, sql from sqlite_master where sql like '%CREATE TABLE%%user_id%'
+			and type = 'table';
+			```
+	*/
+	sql := `
+	UPDATE team_member
+	SET user_id = ?
+	WHERE user_id = ?;
+
+	UPDATE user_role
+	SET user_id = ?
+	WHERE user_id = ?;
+	`
+	_, err := sess.Exec(sql, intoUser.ID, fromUser.ID)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// MergeUser sets the user Server Admin flag
+func (ss *SQLStore) MergeUser(ctx context.Context, intoUserId int64, fromUserIds []int64) error {
```

Comment:
Can this be a Noop for this PR while we figure out the display and conflict logic?

Reply:
this is going to be move to the next PR and kept in the `commands` package instead of having it in `sqlstore` package